src/ts/state/reducer.ts

```diff
@@ -157,6 +157,8 @@ export interface IPluginState<
   docChangedSinceCheck: boolean;
   docIsEmpty: boolean;
   typerighterEnabled: boolean;
+  // the percentage of blocks that have been processed by the matcher service
```

Comment:
Pedantically, that isn't quite right – at the moment, it happens to be the percentage of the number of jobs that have been completed, but this is an implementation detail that might change. The variable name does a good job here, which makes the below suggestion a bit redundant, but more accurate I think – alternatively the comment is perhaps not needed.

```suggestion
  // the percentage of the request that has been processed by the matcher service
```